kim_cnn/args.py

```diff
@@ -0,0 +1,31 @@
+import os
+
+from argparse import ArgumentParser
+
+def get_args():
+    parser = ArgumentParser(description="Kim CNN")
+    parser.add_argument('--no_cuda', action='store_false', help='do not use cuda', dest='cuda')
+    parser.add_argument('--gpu', type=int, default=0) # Use -1 for CPU
```

Comment:
Should this comment be updated? since we need to use `--no_cuda` to actually run on CPU?